arrows/ceres/tests/test_reprojection_error.cxx

```diff
@@ -170,11 +174,9 @@ TEST_P(reprojection_error, compare_projections)
 
 // ----------------------------------------------------------------------------
 #define DISTORTION( t, k ) \
-  reprojection_test{ #t, kwiver::arrows::ceres::t, k }
+  reprojection_test{ #t, kwiver::arrows::mvg::t, k }
 
-INSTANTIATE_TEST_CASE_P(
-  ,
-  reprojection_error,
+INSTANTIATE_TEST_CASE_P(, reprojection_error,
```

Comment:
This looks wrong (non-compliant indent), though obviously the empty macro argument is part of the problem. I don't have time to look right now, but is there any token that can be written there (even a C-style comment?) so there isn't a completely missing argument? (Ideally, I'd probably like to otherwise retain the prior formatting.)

Reply:
reverted to the prior formatting